rsyslog/init.sls

```diff
@@ -25,8 +25,13 @@ config_{{ rsyslog.config }}:
 {% for filename in salt['pillar.get']('rsyslog:custom', {}) %}
 rsyslog_custom_{{filename}}:
   file.managed:
+    - source: salt://rsyslog/files/{{ filename }}
+    {% if filename.endswith('.jinja') %}
+    - template: jinja
+    - name: {{ rsyslog.custom_config_path }}/{{ filename|replace(".jinja", "") }}
+    {% else %}
     - name: {{ rsyslog.custom_config_path }}/{{ filename }}
```

Comment:
You can probably make the name lines the same... The replace just won't do anything in the non-jinja case.